### ISSUE TITLE
Remove default State of Tas on Dates & Times

### ISSUE THREAD

**Issue by Alex-DTO** [AusDTO/gov-au-beta#370]
Current default is TAS simply because that was the only state we had data for initially.

Please could you remove the default option - the first to display should be the first in the list.

I would like to try to avoid the solution being 'Set to NSW' as we may well change the order of tabs in the near future

![image](https://cloud.githubusercontent.com/assets/20697521/17355005/c9bf2aca-5990-11e6-8a29-53ecea71788c.png)

**Comment by catherineedwards** [comment]
@Alex-DTO I am reluctant to add this to our backlog as it has been described at this point.

1. The abbreviations on the tabs have been applied inconsistently - NSW -> Vic. -> Qld, etc. Suggest that we have ACT | NSW | NT | QLD | SA | WA applied.

2. The solution you have prescribed will not accommodate changing requirements in the future. Suggest that we have all public holidays listed, and then filter by state or territory upon selection of the header.

Let's discuss before we add this to the backlog.

**Comment by maadonna** [comment]
That's the correct way to sequence and name the states (until we decide otherwise to make a global content rule). Please don't do them alphabetically.

**Comment by maadonna** [comment]
We should have a proper discussion about design and implementation of tabs before we go any further with this. We need to decide whether tabs are separate pages/URLs (I think no); and, if yes, how would authors indicate they are a set, can they have child pages etc. Need to design good small-screen navigation. And also need to discuss when it is appropriate to even use tabs.

**Comment by klepas** [comment]
> We should have a proper discussion about design and implementation of tabs before we go any further with this. We need to decide whether tabs are separate pages/URLs (I think no); and, if yes, how would authors indicate they are a set, can they have child pages etc. Need to design good small-screen navigation. And also need to discuss when it is appropriate to even use tabs.

There are a non-zero number of design elements that have been requested of the uikit team which I feel haven’t been scoped and signed off by our designers and info architects.

I’m keen to have these discussions heard.

**Comment by streetdaddy** [comment]
@maadonna I read your comment about the sequence, and wondered if the list is ordered by population size?  If it is, WA comes before SA :neckbeard:

**Comment by maadonna** [comment]
Hmmmm....that's interesting. I used to work at the ABS so had the sequence memorised. They still do SA before WA, but that's not logical. We should be logical and switch them. Sorry for not picking that up.

**Comment by streetdaddy** [comment]
@maadonna not at all, kind of random that I checked to be honest.

**Comment by Alex-DTO** [comment]
@streetdaddy look at you, demoting your home SA by accident ;)

@catherineedwards would you be able to inquire with the devs around the request? The precise requirement is that in a page containing tabbed content, the first of these tabs should display by default. At present that will be the case whether or not it's Dates and Times or anything else.

The backlog has a geo-locate feature but we need to alter the as-is to account for the fact that will likely not happen in the immediate future.

@maadonna - looks like we also need to request the SA / WA switch over

**Comment by maadonna** [comment]
Yes, please switch the sequence. I think this is the only place we've done it so far. I'll also let the content guide folks know.

And we still need to have a discussion about the implementation of tabs, and whether they break up the content on a single page; or combine a set of pages (the latter is not standard practice and means the tabbed 'page' can never have child pages).

**Comment by streetdaddy** [comment]
TL;DR; Hold your horses, let's not be to hasty about changing the tab order 🐴 

Since I was destined to become, 'that guy who changed the generally accepted logical ordering of Australian States and Territories', I thought I'd better do my homework.  It's worth noting that the [Australian Bureau of Statistics](http://www.abs.gov.au/ausstats/abs@.nsf/mf/3101.0) orders them the same as we do in the tabs, even though both WA and ACT have a higher population than their direct predecessors, SA and NT respectively, according to December 2015 statistics.

WA's population estimate surpassed SA's in June 1982, according to [Australian Demographics Statistics Quarterly September 1982](http://www.ausstats.abs.gov.au/ausstats/free.nsf/0/BFA101829852D7EACA2576EF0011BD58/$File/31010_09_1982.pdf).  But I couldn't easily find any records going back far enough to confirm when, or even _if_, the ACT's population overtook the NT at some point in history.  In 1911, when both the NT and (A)FCT were created, Canberra already had a higher population than Darwin. However, one might surmise that given the NT's relative landmass, it's territorial population was in fact greater than the ACT, and this has changed at some point in the 70 year period for which I am far to tired to bother trying to find official statistics for.  So the proof still eludes me.

I reviewed the [Territorial evolution of Australia](https://en.wikipedia.org/wiki/Territorial_evolution_of_Australia), to see if there was any other clues.  For a moment I thought it might be ordered by the foundation dates of each state, but WA was renamed from it's former title of Swan River Colony four years prior to South Australia's proclamation, so that theory is out.

At this point, I am rabidly curious to know from whom or whence this generally accepted ordering of Australian States and Territories has come.  However, I am understandably trepidatious about exposing myself to the potential ridicule that may befoul me if I hastily prance into the town square shouting out my discovery to the roof tops, tantalised by the heady tempest of Australia's celebrity circles.  Unless said details can be surfaced and validated by appropriately knowledgeable parties, or currently undiscovered official statistics are uncovered to validate my population theory, I request that the aforementioned change request be chucked in the esky until such time that this little guy says :shipit: indubitably.

**Comment by maadonna** [comment]
I think I have it puzzled out. It's population order by state as at 1901 (Federation). For NT & ACT I think it's population order at 1911 which is when the first Australian census was taken.

**Comment by Alex-DTO** [comment]
Well I'm glad we've got to the bottom of it!? A classic example of where the status quo has become accepted for so long that we've completely forgotten why we do something, such as eating chocolate at easter, why shoe laces are more universally accepted than velcro, or why England's still in Europe.

The only way to settle this is through a referendum. In the spirit of returning our thinking back to 1901 and 1911 when the original rules were thought up. I would bring in a collection of straws (made of straw, not plastic straws) but that's not really in keeping with the rules. Instead we should stand around a print off of the States in their current status quo order, ask who stands for it as it is, and if we hear more than 50% calls of "Here, here" then the current order will be dispatched post haste, and we shall dream up some magical new order of States. Perhaps alphabetical... if that's taken off yet.

Or we get some user research done

Either way... this callout was just to remove the default tab being the 6th tab in the series of 8.... can we not still remove the default?!

**Comment by catherineedwards** [comment]
> Either way... this callout was just to remove the default tab being the 6th tab in the series of 8.... can we not still remove the default?!

I squirm uncomfortably when a solution is requested without a requirement (i.e. what is the problem we are trying to solve?). I also don't know what happens to the page when we "remove the default." I also do not like allocating development time to piecemeal changes that are more than likely going to be overwritten in the future. We should be prioritising analysis on the requirement to inform the solution (i.e. user research).

OR If you can fit a referendum into a sprint, be my guest! :sparkles:

**Comment by Alex-DTO** [comment]
Hey, I'm back on this!

Solution callout was more for a 'just in case' thought - I don't 'want' NSW to be the default, because if we change the order of the States (which I hope we will after some usability testing) then we're back to square 1 with the 2/3rd tab being opened by default.

The actual need is that a user should be presented data from the top of the listing, and navigate below that if they require.

FYI we will be looking to have a new set of data for Public Holidays with an 'All Australia' tab - this will likely be the first tab, so again would be the default. At present School Holidays looks like it will stay as it, Timezones may or may not change

**Comment by ltankey** [comment]
At the risk of reducing the conversation in this thread to bike-shedding - @Alex-DTO would it be fair to say "tabs exist in some order" and that convention and commonsense would suggest that the first tab is the one that is shown by default? 

I don't think we need to be particularly concerned with "which tab should show first" when convention is that it's usually the first.

**Comment by Alex-DTO** [comment]
Totes ;)

**Comment by tessereth** [comment]
For now, I've changed the default to NSW because it's just a data change. Once we properly implement times and dates, we can make the default the first tab. I believe that solves this issue (from a platform standpoint at least).
